src/bundler.ts

```diff
@@ -130,6 +134,37 @@ export class Bundler {
   }
 
   /**
+   * Generates a BundleManifest with all bundles defined, using entrypoints,
+   * strategy and mapper.
+   *
+   * @param {Array<string>} entrypoints The list of entrypoints that will be
```

Comment:
I'm all for leveraging the types from the TypeScript.  I've removed the type definitions.  Syntax highlighting for jsdoc starts to disappear so I'm using the hyphen to separate param name from description since it's valid jsdoc according to http://usejsdoc.org/tags-param.html and reads easier.